SynNoMoreEasyEnemies/Program.cs

```diff
@@ -141,21 +143,21 @@ out LazySettings
             foreach (var achr in state.LoadOrder.PriorityOrder.OnlyEnabled().PlacedNpc().WinningContextOverrides(state.LinkCache)) {
                 // If the ACHR has no base, skip it
                 // Not even sure this is possible
-                if (achr.Record.Base == null) { continue; }
+                if (achr.Record.Base.IsNull) continue;
```

Comment:
Just based on their typing and choices I made for what could be null and how it's exposed.  If you hover over `achr.Record.LevelModifier`, you'll see it's `Level?`, where `?` means it might be null and throw an exception if you do something with it when it is.  So you need to check.

compared to `achr.Record.Base` which is of type `IFormLinkNullableGetter<INpcGetter>`.   That signature means its an object that cannot be null in the way that it would throw a NullReferenceException if you were to interact with it.  This is a choice of exposure that I made for very gritty specific reasons I can get into if you care. In a previous Mutagen version, it used to be exposed as `IFormLinkGetter<INpcGetter>?` as its API for exposing when it was null.  If it was still that way, it would need a `!= null` check on the `achr.Record.Base` field itself.  But as it is now, the link field as an object will never be null.

However, FormID/FormKeys can and do point to no record by having all zeros.  That's what we want to check.  The link object is never null, but the pointer (FormID) might be.  So that's why we call `link.IsNull` instead of `link != null`.  We know `link` object is never null, but we want to know if it's pointing to nothing or not.